lib/content-services/src/lib/tag/tags-creator/tags-creator.component.html

```diff
@@ -1,54 +1,38 @@
 <div class="adf-tags-creation">
+    <div *ngIf="tagNameControlVisible" class="adf-tag-name-field">
+        <div class="adf-tag-search-field">
```

Comment:
Do we need this div? It contains only input so can we add this class to input directly and remove that div?